src/components/graph/menus/create-node-menu.jsx

```diff
@@ -226,7 +227,7 @@ const CreateNodeMenu = ({
             id: 'buildNode',
             disabled:
                 activeNode?.data?.globalBuildStatus?.startsWith('BUILT') ||
-                activeNode?.data?.globalBuildStatus === 'BUILDING' ||
+                activeNode?.data?.globalBuildStatus === BUILD_STATUS.BUILDING ||
```

Comment:
> There's also an occurrence of this in `gridstudy-app/src/components/graph/network-modification-tree-model.js`

Yeah, have miss some because it wasn't linked to the Redux store directly.


> Also valid for several uses of `'BUILT'` in this component and `gridstudy-app/src/components/graph/util/model-functions.js` which could be replaced by `BUILD_STATUS.BUILT`

I only see one use in this file, and that's normal because it test 3 potentials status:
```typescript
export enum BUILD_STATUS {
    NOT_BUILT = 'NOT_BUILT',
    BUILDING = 'BUILDING',
    BUILT = 'BUILT',
    BUILT_WITH_WARNING = 'BUILT_WITH_WARNING',
    BUILT_WITH_ERROR = 'BUILT_WITH_ERROR',
}
```